### PR TITLE
Server refuses to start if max_open_connections is below 5

### PR DESCRIPTION
**Pull request by Gabrielopesantos**
# Description
Proposal to fix issue #2095

### CODE REVIEW

File: internal/db/db.go [hunk 1]
```diff
@@ -95,6 +95,9 @@ func Open(dbType DbType, connectionUrl string, opt ...Option) (*DB, error) {
 		wrappedOpts = append(wrappedOpts, dbw.WithLogger(opts.withGormFormatter))
 	}
 	if opts.withMaxOpenConnections > 0 {
+        if opts.withMaxOpenConnections > 0 && opts.withMaxOpenConnections < 5 {
```
Comment (johanbrandhorst) [inline internal/db/db.go:98]: This first check is unnecessary, we already know this at this point
```suggestion
        if opts.withMaxOpenConnections < 5 {
```

**Review by johanbrandhorst — CHANGES_REQUESTED** [review]
Hi @Gabrielopesantos, thanks for your contribution! I've just got one minor suggestion, otherwise this looks great!

File: internal/db/db.go (hunk 1 above)
Comment (Gabrielopesantos) [reply internal/db/db.go]: Thanks for the feedback.
Yes, my bad. If statement at line 97 which covers it.

**Review by johanbrandhorst — APPROVED** [review]
LGTM!

**Comment by johanbrandhorst** [comment]
I'm going to try closing and reopening this myself and see if it will trigger our CI jobs.

**Comment by johanbrandhorst** [comment]
It appears there is no way for me to run our CI tests on this PR. I cloned the PR and ran the tests locally and they pass, so I'm going to merge this.

**Comment by johanbrandhorst** [comment]
Thank you for your contribution!

**Pull request merged**